models/experimental/functional_mobilenetv2/test/test_ttnn_mobilenetv2.py

```diff
@@ -0,0 +1,58 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+
+import pytest
+import ttnn
+import torch
+
+from tests.ttnn.utils_for_testing import assert_with_pcc
+
+from models.experimental.functional_mobilenetv2.reference.mobilenetv2 import Mobilenetv2
+from models.experimental.functional_mobilenetv2.tt.model_preprocessing import (
+    create_mobilenetv2_input_tensors,
+    create_mobilenetv2_model_parameters,
+)
+from models.experimental.functional_mobilenetv2.tt import ttnn_mobilenetv2
+import os
+from models.utility_functions import (
+    skip_for_grayskull,
+)
+
+
+@pytest.mark.parametrize("device_params", [{"l1_small_size": 32768}], indirect=True)
+@skip_for_grayskull()
+def test_mobilenetv2(device, reset_seeds):
+    if not os.path.exists("models/experimental/functional_mobilenetv2/mobilenet_v2-b0353104.pth"):
+        os.system(
+            "bash models/experimental/functional_mobilenetv2/weights_download.sh"
+        )  # execute the weights_download.sh file
+
+    state_dict = torch.load("models/experimental/functional_mobilenetv2/mobilenet_v2-b0353104.pth")
+    ds_state_dict = {k: v for k, v in state_dict.items()}
+    torch_model = Mobilenetv2()
+
+    new_state_dict = {}
+
+    for (name1, parameter1), (name2, parameter2) in zip(torch_model.state_dict().items(), ds_state_dict.items()):
+        if isinstance(parameter2, torch.FloatTensor):
+            new_state_dict[name1] = parameter2
+
+    torch_model.load_state_dict(new_state_dict)
+    torch_model.eval()
+    torch_input_tensor, ttnn_input_tensor = create_mobilenetv2_input_tensors()
```

Comment:
demo should support feeding real input images, even if it is not tested in CI?

have you tested it with real images? pcc=0.95 is decent result, but the best passing criteria is to check real images.

Reply:
@uaydonat Added support to feed real images as imput in demo.py